src/main/java/com/google/api/codegen/py/PythonImportHandler.java

```diff
@@ -233,6 +233,17 @@ public String addImportLocal(String moduleName, String attributeName) {
     return addImport(ImportType.APP, moduleName, attributeName).shortName();
   }
 
+  /** Add an import for the proto associated with the given type. */
+  private PythonImport addImportForType(TypeRef type) {
```

Comment:
Make this method `addImportForMessage(MessageType message)`. As-is, this will raise a NullPointerException for any non-message TypeRef.